source/templates/env_vars.slim

```diff
@@ -10,13 +10,21 @@
 		article
 			ul.env-vars[order-by-drag models="envVarsOfSource.envVars"]
 				li[ng-repeat="envVar in envVarsOfSource.envVars"]
-					form[name="envVarForm"]
+					p.progress-indicator[progress-model="envVar.customProperties.progress"]
+					form[name="envVarForm" ng-if="!envVar.customProperties.progress.isInProgress"]
 						.key-value-delete
-							input.key[type="text" name="key" ng-model="envVar.key" ng-model-options="{getterSetter: true}" custom-validators="{'key': Variable.isValidKey(envVar.key())}" placeholder="#{ data[:strings][:env_vars][:key_placeholder] }" skip-drag-order]
-							textarea.value[name="value" msd-elastic ng-model="envVar.value" ng-model-options="{getterSetter: true}" ng-trim="false" placeholder="#{ data[:strings][:env_vars][:value_placeholder] }" skip-drag-order]
-							button.delete-env-var.rebo.danger[type="button" ng-click="envVarsCtrl.removeEnvVarFromEnvVars(envVar, envVarsOfSource.envVars)"] = svg("cross")
-						.is-expand
-							checkbox.small[name="isExpand" ng-model="envVar.isExpand" ng-model-options="{getterSetter: true}" checkbox-id-expression="'env-var-editor-is-expand-checkbox-' + $index"]
-							label[for="env-var-editor-is-expand-checkbox-{{ $index }}"] == data[:strings][:env_vars][:replace_variables_label]
+							input.key[type="text" name="key" ng-model="envVar.key" ng-model-options="{getterSetter: true}" ng-disabled="!envVar.isKeyChangeable" custom-validators="{'key': Variable.isValidKey(envVar.key())}" placeholder="#{ data[:strings][:env_vars][:key_placeholder] }" skip-drag-order]
+							.value-with-actions
```

Comment:
Not looking ok on small screen:

![screen shot 2018-02-08 at 20 13 21](https://user-images.githubusercontent.com/8428264/35993089-8912c820-0d0c-11e8-83e9-79c59b620e85.png)